modules/tinymce/src/plugins/paste/main/ts/api/Settings.ts

```diff
@@ -59,6 +59,8 @@ const getForcedRootBlock = (editor: Editor) => editor.getParam('forced_root_bloc
 
 const getForcedRootBlockAttrs = (editor: Editor) => editor.getParam('forced_root_block_attrs');
 
+const getTabSpaces = (editor: Editor) => editor.getParam('paste_tab_spaces', 4, 'number');
```

Comment:
Add this to the settings interface also? Or do we not do that because it's a plugin setting

Reply:
Yeah, only core settings go in that file.